feat(suspect-tags): add to transaction summary (#3469)
getsentry/sentry-docs: src/docs/product/performance/transaction-summary.mdx

_Note_: Currently, only transaction data - the transaction name and any attributes the transaction inherits from its root span - is searchable. Data contained in spans other than the root span is not indexed and therefore cannot be searched.

> ## Suspect Tags

Reviewer: Should we mention this is an EA feature?

Reviewer 2: If it is, then, yes. Here's our standard text, Dora:
```suggestion
## Suspect Tags

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>
```

_Note_: Currently, only transaction data - the transaction name and any attributes the transaction inherits from its root span - is searchable. Data contained in spans other than the root span is not indexed and therefore cannot be searched.

## Suspect Tags

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

For this transaction, we'll list out suspect tags that often correspond to slower transactions. By default, we sort tags by the total time lost. Additional information includes:

- **Tag Key** is the tag category (ex. device, geo, etc.)
- **Tag Value** is the specific tag value (ex. android, south america, etc.)
- **Avg (Tag) Duration** provides the average duration for a transaction with this tag
- **Comparison to Avg** notes how much slower or faster this tag is than the average duration of this transaction

> - **Total Time Lost** calculates how much total time you've lost in the selected time range when comparing the average tag duration v. the average transaction duration

Reviewer: ```suggestion
- **Total Time Lost** calculates how much total time you've lost in the selected time range when comparing the average tag duration to the average transaction duration
```

> For this transaction, we'll list out suspect tags that often correspond to slower transactions. By default, we sort tags by the total time lost. Additional information includes:
> 
> - **Tag Key** is the tag category (ex. device, geo, etc.)
> - **Tag Value** is the specific tag value (ex. android, south america, etc.)
> - **Avg (Tag) Duration** provides the average duration for a transaction with this tag
> - **Comparison to Avg** notes how much slower or faster this tag is than the average duration of this transaction
> - **Total Time Lost** calculates how much total time you've lost in the selected time range when comparing the average tag duration v. the average transaction duration

Reviewer 3: ```suggestion
The transaction summary includes a list of suspect tags that often correspond to slower transactions. By default, we sort tags by the total time lost. The list includes additional information:

- **Tag Key**: The tag category (for example, device, geo)
- **Tag Value**: The specific tag value (for example, Android, South America)
- **Avg (Tag) Duration**: Provides the average duration for a transaction with this tag
- **Comparison to Avg**: Indicates how much slower or faster this tag is than the average duration of this transaction
- **Total Time Lost**: Calculates how much total time you've lost in the selected time range when comparing the average tag duration to the average transaction duration
```

> To investigate why these tags are causing you problems, click on the tag value to filter this transaction summary further.

Reviewer 3: ```suggestion
To investigate why these tags are affecting performance, click on the tag value to filter the transaction summary further.
```
